src/test/java/com/uber/cadence/workflow/WorkflowTest.java

```diff
@@ -6118,6 +6120,19 @@ private TracingWorkflowInterceptor(FilteredTrace trace, WorkflowInterceptor next
       this.next = Objects.requireNonNull(next);
     }
 
+    /**
+     * @param workflowDefinition
+     * @param input
+     * @return
+     * @throws CancellationException
+     * @throws WorkflowExecutionException
+     */
+    @Override
+    public byte[] executeWorkflow(SyncWorkflowDefinition workflowDefinition, byte[] input)
```

Comment:
Instead of using raw input, let's create an entity to include WorkflowStartEventAttributes.